### PR TITLE
[Bugfix] Fix issue with labels for zero point exercises with bonus

### PR DESCRIPTION
**Pull request by fde312**
### Checklist
- [x] I tested *all* changes and *all* related features locally.
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).
- [x] Client: I added multiple screenshots/screencasts of my UI changes

### Motivation and Context

Follow-up of #2539, issue with wrong point labels.
### Description

- added `!= undefined`
### Steps for Testing

1. Log in to Artemis
2. Navigate to Course Administration
3. Create Exercise with 0 points and with e.g. 10 bonus points
4. Check that for the exercise detail view and exercise participation view the points are displayed correctly (e.g 0 (Bonus: 10) -> see screenshots below)
Exam exercises:
1. Create exam with same exercise setup as above (you can also import them) and also with 0 points and 0 bonus points
2. Participate and set the result release date of the exam 
3. Check Summary table that 0 points and 0 bonus points are displayed with value 0 (see screenshot below)

### Test Coverage

Nothing changed.

### Screenshots

exercise detail view:
image

exam summary view:
image

exercise participation view:
image

### CODE REVIEW

**Review by airelawaleria — CHANGES_REQUESTED** [review]
Issues with modelling and text exercises (0 normal points, 10 bonus)
Tested with @simonzilker @JohannesWeiss @marvinchimeka @LudwigWeide @EneaGore on ts1
image

**Review by marvinchimeka — CHANGES_REQUESTED** [review]
Testing in testing session. .Agreeing with @airelawaleria's statement.

**Review by simonzilker — CHANGES_REQUESTED** [review]
Tested in testing session. Can confirm what @airelawaleria said!

**Review by JohannesWeiss — CHANGES_REQUESTED** [review]
Tested in testing session, can confirm the issue from @airelawaleria  with text/ model exercises (normal and in exams). Code looks good, though.

**Review by LudwigWeide — CHANGES_REQUESTED** [review]
Tested in testing session. Same issue as @airelawaleria .

**Review by EneaGore — COMMENTED** [review]
Same issue as @airelawaleria .

File: src/main/webapp/app/exam/participate/summary/points-summary/exam-points-summary.component.html [hunk 1]
```diff
@@ -20,10 +20,10 @@ <h4>{{ 'artemisApp.exam.examSummary.points.overview' | translate }}</h4>
                     {{ calculateAchievedPoints(exercise) }}
                 </td>
                 <td>
-                    <span *ngIf="exercise.maxScore">{{ exercise.maxScore }}</span>
+                    <span *ngIf="exercise.maxScore != undefined">{{ exercise.maxScore }}</span>
```
Comment (ivanchimeno) [inline src/main/webapp/app/exam/participate/summary/points-summary/exam-points-summary.component.html:23]: I might be wrong but how about we change that: `*ngIf="exercise.maxScore"`? The check would cover not only null/undefined but also wouldn't display if max score is 0

**Review by airelawaleria — APPROVED** [review]
The previous request for changes was unrelated to this pr. Re-tested the labels, works fine :)

**Review by marvinchimeka — APPROVED** [review]
Approve. The mentioned request about changes was unrelated to this pr.

**Review by LudwigWeide — APPROVED** [review]
Approved in testing session with @airelawaleria .

**Review by simonzilker — APPROVED** [review]
Approve!

**Review by EneaGore — APPROVED** [review]
Approve

**Review by JohannesWeiss — APPROVED** [review]
Approve, same reason as mentioned above.

**Review by krusche — APPROVED** [review]

**Pull request merged**